src/vkvg_context.c

```diff
@@ -987,6 +994,13 @@ void vkvg_paint (VkvgContext ctx){
 	_ensure_renderpass_is_started (ctx);
 	_draw_full_screen_quad (ctx, true);
 }
+
```

Comment:
I guess the vkvg_fill line 990 must also be wrapped in a private method.

Reply:
Oh right.

Reply:
Done.